### ISSUE TITLE
`yank` docs are unclear

### ISSUE THREAD

**Issue by jasonschroeder-sfdc** [buildpacks/docs#493]
**On Page:** [pack buildpack yank](https://buildpacks.io/docs/tools/pack/cli/pack_buildpack_yank/)

What does it *do*? It yanks the buildpack. Is it the opposite of `pack buildpack register` ?

**Comment by hone** [comment]
Not exactly, it removes it from the index. I got this term originally from [rubygems](https://guides.rubygems.org/command-reference/#gem-yank).

**Comment by samj1912** [comment]
Similar term also exists for PyPI https://pypi.org/help/#yanked
